specification/help/resource-manager/readme.az.md

```diff
@@ -0,0 +1,28 @@
+## AZ
+
```

Comment:
Please remove readme.az.md  and readme.cli.md,  those files are not longer needed.

Reply:
Removed the files.